.github/workflows/update-on-version.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
       - name: Generate documentation and assets
         if: steps.version-update.outputs.version_changed == 'true'
-        run: make
+        run: make update-helm-readme
```

Comment:
Hey @ibakshay thanks for working on that.

I think makes sense run both commands: fmt-docs update-helm-readme

Regarding the update-helm-readme, IIRC it depends on docker, do we need to change anything on CI to run it properly? I mean on this update-on-version action.

Reply:
You're right mate, thanks for working on this. 🥳